README.md

```diff
@@ -20,22 +18,19 @@ Print or pdf generation buttons for generating Tignum reports
         background-color: #3d6fb4;
     }
```

Comment:
Add info about the css variables : that --my-element-text-color, is nowhere documented

Reply:
Updated on README